package.json

```diff
@@ -3,20 +3,23 @@
 	"version": "4.1.5",
 	"description": "A very fast HTML parser, generating a simplified DOM, with basic element query support.",
 	"main": "dist/index.js",
-	"module": "dist/esm/index.js",
 	"types": "dist/index.d.ts",
 	"scripts": {
-		"test": "mocha",
+        "compile": "tsc",
+        "build": "npm run lint && npm run clean && npm run compile:cjs && npm run compile:amd",
+        "compile:cjs": "tsc -m commonjs",
+        "compile:amd": "tsc -t es5 -m amd -d false --outFile ./dist/main.js",
+        "---------------": "",
+		"test": "mocha ./test/tests/**/*.js",
+		"test:src": "cross-env TEST_TARGET=src mocha ./test/tests",
+		"test:dist": "cross-env TEST_TARGET=dist mocha ./test/tests",
+        "benchmark": "node ./test/benchmark/compare.mjs",
+        "---------------  ": "",
 		"lint": "eslint ./src/*.ts ./src/**/*.ts",
 		"clean": "del-cli ./dist/",
-		"ts:cjs": "tsc -m commonjs",
-		"ts:amd": "tsc -t es5 -m amd -d false --outFile ./dist/main.js",
-		"ts:esm": "tsc -t es2019 -m esnext -d false --outDir ./dist/esm/",
-		"build": "npm run lint && npm run clean && npm run ts:cjs && npm run ts:amd && npm run ts:esm",
-		"dev": "tsc -w & mocha -w ./test/*.js",
-		"pretest": "tsc -m commonjs",
-		"release": "yarn build && np",
-		"prepare": "npm run build"
+        "--------------- ": "",
+        "posttest": "yarn run benchmark",
+		"prepare": "cd test && yarn install"
```

Comment:
why remove the `npm run build` prepare script?
this is required for "install from git", `cd test && yarn install` is not